core/drivers/clk/sam/clk-sam9x60-pll.c

```diff
@@ -0,0 +1,517 @@
+// SPDX-License-Identifier: GPL-2.0+
+/*
+ *  Copyright (C) 2019 Microchip Technology Inc.
+ *
```

Comment:
can remove this empty inline comment line.

Reply:
Removed the empty line and updated the license model, thanks.